src/main/java/networkbook/model/person/Specialisation.java

```diff
@@ -9,15 +9,23 @@
  */
 public class Specialisation {
 
-    public static final String MESSAGE_CONSTRAINTS = "Specialisations can take any values, and it should not be blank.";
+    public static final String MESSAGE_CONSTRAINTS =
+            "Specialisations can take any value, but should not be blank.\n"
+            + "Additionally, only the first letter of each word may be uppercase,\n"
+            + "and more than one space between each word is not allowed";
 
     /*
      * The first character of the specialisation must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
+     * Proceeding letters of a word should not be uppercase, unless...
      */
-    public static final String VALIDATION_REGEX = "[^\\s].*";
+    public static final String NOT_ALL_CAPITALIZED_REGEX = "^[A-Za-z0-9][a-z0-9.,_-]{0,99}$";
+    /*
+     * The word is in all uppercase, such as for acronyms are also acceptable.
+     */
+    public static final String ALL_CAPITALIZED_REGEX = "^[A-Z]+$";
 
-    public final String value;
+    public final String specialisation;
```

Comment:
Please make sure that the value is not publicly accessible. Please use a getter function only.